Makefile

```diff
@@ -14,15 +16,15 @@ CPPFLAGS+=	-DVERSION=\"${VERSION}\" -DYANKCMD=\"${YANKCMD}\"
 all: yank
 
 yank: yank.c
-	${CC} ${CFLAGS} -o $@ $< ${LDFLAGS} ${CPPFLAGS}
+	${CC} ${CFLAGS} -o ${PROG} $< ${LDFLAGS} ${CPPFLAGS}
```

Comment:
Cange `$<` to `yank.c`. I'm running into to the following error when applying my
suggested changes:

```
Using $< in a non-suffix rule context is a GNUmake idiom (Makefile:19)
````